src/main/resources/templates/error/4xx.mustache

```diff
@@ -0,0 +1,17 @@
+{{>layout/header}}
+<div class="container">
+    <div class="default-error">
+        <div class="error_wrapper">
+            <img src="images/icon-error.svg" loading="lazy" alt="" class="error_icon">
+            <div>
+                <h1 class="error_title">{{errorTitle}}</h1>
+                <p>{{errorDescription}}
+                    <a href="{{errorReportEmailLink}}">
+                        {{errorReportEmail}}
+                    </a>
+                </p>
+            </div>
+        </div>
+    </div>
+</div>
+{{>layout/footer}}
```

Comment:
Shouldn't the `errorNotification` template be reused here since it has the same appearance?

Reply:
now it's updated to use the `errorNotification`